.github/workflows/scripts/win_build.ps1

```diff
@@ -20,7 +20,7 @@ if (-not (Test-Path $libsDir)) {
 
 $RepoURL = 'https://github.com/taichi-dev/taichi'
 
-SetupCCacheLocal "$libsDir/ccache"
+# SetupCCacheLocal "$libsDir/ccache"
```

Comment:
Why commented out?

Reply:
Trying to disable ccache to confirm if the fail is caused by using old llvm binary